web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
I actually see no `.dropdown-submenu` elements in the page. Is this really used for anything? Can't you just stick to the built-in [BS dropdowns](http://getbootstrap.com/javascript/#dropdowns)?